add aggregate documentation

lboeman opened the pull request.

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,78 @@ Create New Site
+### Add Observations to an Aggregate
+{: .anchor}
+
+Each aggregate is defined by the observations it contains. Before you can add
```
Observations must be defined before they can be added to an aggregate (see Create New Observation)

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,78 @@ Create New Site
+
+   <img class="my-3" src="/images/aggregate_listing.png"/>
+
+2. Enter the metadata for your Aggregate.
```
might want to mention restrictions on interval length and variable here

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,78 @@ Create New Site
+Create New Aggregate
+--------------------
+{: .anchor}
+
```
I think we need to say 1 or 2 sentences about how aggregates work. First define metadata, then associate *preexisting* observations with it and/or create *new* forecasts for it.

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,78 @@ Create New Site
+2. Click the **Add Observation** button.
+   <img class="my-3" src="/images/aggregate_page.png"/>
+
+3. Enter an **Effective From** date. This will determine the start of the period
```
need to describe what happens if you enter an effective from that is too early or late

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,78 @@ Create New Site
+
+Observations are removed from an aggregate by setting their "Effective Until" date.
+Any observation with an "Effective Until" date that has passed will not be included
+when computing the values for the aggregate.
```
Is this correct? I thought new data would not be included in the aggregate, but data between effective from and until would still be.

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,78 @@ Create New Site
+
+   <img class="my-3" src="/images/aggregate_page_with_obs.png"/>
+
+3. Enter the "Effective Until" date for the observation. This will determine the end
```
To remove it entirely, set "Effective until" to be earlier than "Effective from"?

lboeman commented:
@wholmgren You're right, we didn't ever document the ability to create a forecast/obs from the forecasts or observations page.  I think I've incorporated all of your feedback and updated the images to reflect the latest version of the dashboard.  Rendered at https://lboeman.github.io/documentation/dashboard/#create-new-aggregate

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,94 @@ Create New Site
+how aggregation will be performed and the characteristics of the resulting
+timeseries data.
+
+Observations are included in an aggregate between an *Effective From* and
```
This paragraph seems out of place because these options are not exposed in the aggregate creation form. Move to Add Observations to an Aggregate?

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,94 @@ Create New Site
+### Add Observations to an Aggregate
+{: .anchor}
+
+Aggregates are defines by the observations it contains. Observations must be
```
Change "Aggregates are defines by the observations it contains. Observations must be" to... Observations may be added to an Aggregate by following the steps below. Observations must already exist in the Arbiter before they...

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,94 @@ Create New Site
+   the left sidebar. Select the Aggregate to add observations to.
+
+2. Click the **Add Observation** button.
+   <img class="my-3" src="/images/aggregate_page_no_obs.png"/>
```
I don't think this screenshot helps here

wholmgren commented on dashboard_documentation.md:
```diff
@@ -69,6 +69,94 @@ Create New Site
+
+   <img class="my-3" src="/images/aggregate_observation_effective_until_form.png"/>
+
+4. After submitting the form, you will be return to the aggregate page.
```
```suggestion
4. After submitting the form, you will be returned to the aggregate page.
```

wholmgren replied:
or just delete this step and image

wholmgren commented on dashboard_documentation.md:
```diff
@@ -95,10 +183,13 @@ To create an Observation or Forecast, an associated site must already exist (see
+
+1.  Navigate to the sites or aggregates page using the sidebar. From the
+    applicable listing page, select the site or aggregate for which you
+    are adding a Forecast.
```
Navigate to the sites or aggregates page using the sidebar, then select the site or aggregate for which you are adding a Forecast.
